oops/backplane/unittester_support.py

```diff
@@ -0,0 +1,466 @@
+################################################################################
+# oops/backplane/unittester_support.py
+################################################################################
+
+import unittest
+import re
+import os
+import sys
+import numpy as np
+
+from polymath   import Scalar
+from PIL        import Image
+
+
+#===============================================================================
+def _read_image(filename):
+    """Save an image file of a 2-D array.
+
+    Input:        filename    the name of the input file, which should end with
+                              the type, e.g., '.png' or '.jpg'
+    """
+    try:
+        im = Image.open(filename, mode="r")
+    except:
+        return(None)
+
+    return(Scalar.as_scalar((np.array(im))))
```

Comment:
return and assert are not functions. They should not have () around their values and need a space after the keyword. Throughout this file.